### PR TITLE
Adding support to integrationnet

### PR DESCRIPTION
**Pull request by marcus-girolneto**
### Changes
+ Adding support to the new network: IntegrationNet

### JIRA
[DEW-77](https://constellationnetwork.atlassian.net/browse/DEW-77)

### Notes
+ We don't have the official URLs yet
+ We are using the URLs of testnet while we don't have the official ones
+ We should remove the tmp file before merging

[DEW-77]: https://constellationnetwork.atlassian.net/browse/DEW-77?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
-import styles from './Footer.module.scss';
 import FooterText from '../../assets/icons/FooterConstellation.svg';
+import styles from './Footer.module.scss';
 
 export const Footer = () => {
+  const handleOpenForm = () => {
+    window.open('https://www.google.com', '_blank');
```
Comment (juandavidkincaid) [inline src/components/Footer/Footer.tsx:6]: *Note:* Pending change to typeform.

Ps. internal note for me to review later.

File: src/components/TransactionsTable/SkeletonTransactionsTable.tsx [hunk 2]
```diff
@@ -59,6 +77,11 @@ const SkeletonTransactionRow = ({ isHomePage, forSnapshots }: { isHomePage: bool
       <div className={styles2.txnCell}>
         <div className={`${styles.skeleton} ${styles.value}`} />
       </div>
+      {showMetagraphSymbol && (
+        <div className={styles2.txnCell}>
+          <div className={`${styles.skeleton} ${styles.value}`} />
```
Comment (juandavidkincaid) [inline src/components/TransactionsTable/SkeletonTransactionsTable.tsx:82]: Please use clsx, or cls, or classnames.

File: src/components/TransactionsTable/TransactionRow.tsx [hunk 3]
```diff
@@ -87,21 +91,21 @@ export const TransactionRow = ({
               </div>
             </div>
           </div>
-          <div className={`${styles.txnCell} ${styles.date} ${styles.timestamp}`}>
+          <div className={`${clsx( isLastRow ? styles.txnCellLastRow :  styles.txnCell)} ${styles.date} ${styles.timestamp}`}>
             <p data-tip={fullDate}>{date}</p>
             <ReactTooltip />
           </div>
-          <div className={`${styles.txnCell}`}>
+          <div className={`${clsx( isLastRow ? styles.txnCellLastRow :  styles.txnCell)}`}>
```
Comment (juandavidkincaid) [inline src/components/TransactionsTable/TransactionRow.tsx:98]: Interpolation is not doing anything here.

File: src/components/TransactionsTable/TransactionRow.tsx [hunk 4]
```diff
@@ -124,7 +128,7 @@ export const TransactionRow = ({
               </div>
             </div>
           </div>
-          <div className={clsx(styles.txnCell, styles.txnDirection)}>
+          <div className={clsx(clsx( isLastRow ? styles.txnCellLastRow :  styles.txnCell), styles.txnDirection)}>
```
Comment (juandavidkincaid) [inline src/components/TransactionsTable/TransactionRow.tsx:131]: Use just one clsx call.

File: .env [hunk 5]
```diff
@@ -6,6 +6,9 @@ REACT_APP_TESTNET_L0_NODE_URL=https://l0-lb-testnet.constellationnetwork.io
 REACT_APP_MAINNET_TWO_BE_URL=https://be-mainnet.constellationnetwork.io
 REACT_APP_MAINNET_TWO_L0_NODE_URL=https://l0-lb-mainnet.constellationnetwork.io
 
+REACT_APP_INTEGRATIONNET_TWO_BE_URL=https://be-testnet.constellationnetwork.io
+REACT_APP_INTEGRATIONNET_TWO_L0_NODE_URL=https://l0-lb-testnet.constellationnetwork.io
```
Comment (juandavidkincaid) [inline .env:10]: Isn't this file generated?, if so, can you please remove it?

**Review by juandavidkincaid — CHANGES_REQUESTED** [review]
In general looks great, with a couple of pattern changes and style changes...

Were you able to configure the domains w/ Alex?

File: .env (hunk 5 above)
Comment (IPadawans) [reply .env]: Done

**Review by juandavidkincaid — APPROVED** [review]

**Pull request merged**